server/hotrod/src/test/java/org/infinispan/server/hotrod/test/RemoteTransaction.java

```diff
@@ -162,14 +162,14 @@ private byte[] get(byte[] key) {
       return performGet(key).value;
    }
 
-   private TxResponse prepare() {
+   public TestResponse prepare() {
```

Comment:
why change `TxResponse` to `TestResponse`?

Reply:
I needed to change if you get an error during prepare it would return a different type which I needed to assert on.